Add YouTube link to README.md (#25)
TYPO3/coding-standards: README.md

* Ensure Concatenation to have at least one whitespace around
* Remove trailing whitespace at the end of blank lines.

## Video Tutorial

> [![TYPO3-Tutorial - The TYPO3 Coding Standards Package](Documentation/Files/youtube-video-tutorial.png)](https://youtu.be/P9fafF2IVpY)

Reviewer: ```suggestion
[![TYPO3-Tutorial - The TYPO3 Coding Standards Package](/Documentation/Files/youtube-video-tutorial.png)](https://youtu.be/P9fafF2IVpY)
```

manually on the `Actions` tab on GitHub by selecting `Core Synchronization` and
running the workflow on the main branch.

## Video Tutorial

> [![TYPO3-Tutorial - The TYPO3 Coding Standards Package](Documentation/Files/youtube-video-tutorial.png)](https://www.youtube.com/watch?v=P9fafF2IVpY)

Reviewer: ```suggestion
[![TYPO3-Tutorial - The TYPO3 Coding Standards Package](Documentation/Files/youtube-video-tutorial.png)](https://youtu.be/P9fafF2IVpY)
```

> [![TYPO3-Tutorial - The TYPO3 Coding Standards Package](Documentation/Files/youtube-video-tutorial.png)](https://www.youtube.com/watch?v=P9fafF2IVpY)

Reviewer: ```suggestion
```

set of rules in 2015, and to the creators and maintainers of PHP-CS-Fixer
package.

## See also

> YouTube: [TYPO3-Tutorial - The TYPO3 Coding Standards Package](https://www.youtube.com/watch?v=P9fafF2IVpY)

Reviewer 2: Maybe we can also add a preview image, e.g., like this: https://github.com/TYPO3-Documentation/tea/blob/main/README.md?plain=1#L33
